tests/infrastructure/golden_images/update_boot_source/utils.py

```
@@ -313,3 +314,48 @@ def wait_for_data_source_unchanged_referenced_volume(data_source: DataSource, vo
                 )
     except TimeoutExpiredError:
         return
+
+
+def wait_for_data_source_updated_referenced_volume(
+    data_source: DataSource,
+    check_exact_match: bool,
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)